Makefile

```diff
@@ -414,6 +415,12 @@ pre-commit-install:
 ensure-relative-imports:
 	./scripts/ensure_relative_imports.sh
 
+.PHONY: streamlit-performance-lighthouse
+# Run Lighthouse performance tests
+streamlit-performance-lighthouse:
```

Comment:
nit: could this just be lighthouse?

Reply:
It could be, yes! However, I was attempting to namespace these, since there will at least also be `streamlit-performance-python` in the future. This naming convention also follows over the to the CI job `Streamlit Performance - Lighthouse` etc.

Reply:
ah okay, fair enough

Reply:
maybe the in-between term here could be `performance-lighthouse` since most other tasks in the makefile are also prefixed with `streamlit-`